packages/flutter/lib/src/services/features.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+/// Whether the multi-window feature  is enabled for the current application.
+///
+/// Do not use this API. Flutter can and will make breaking changes to this API.
+@internal
```

Comment:
Discussion on `@internal` in the main PR: https://github.com/flutter/flutter/pull/168437#discussion_r2093472508